### PR TITLE
Security list item

### PR DESCRIPTION
**Pull request by shaobinspire**

### CODE REVIEW

File: Applications/Spire/Include/Spire/Ui/SecurityListItem.hpp [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+      /** Returns the security. */
+      const Nexus::SecurityInfo& get_security() const;
+
+    private:
+      Nexus::SecurityInfo m_security_info;
+  };
+}
+
+#endif
```
Comment (spireken) [inline Applications/Spire/Include/Spire/Ui/SecurityListItem.hpp:29]: Needs a newline.

**Review by spireken — CHANGES_REQUESTED** [review]
Looks good.

File: Applications/Spire/Include/Spire/Ui/SecurityListItem.hpp (hunk 1 above)
Comment (shaobinspire) [reply Applications/Spire/Include/Spire/Ui/SecurityListItem.hpp:29]: Good catch.

**Review by spireken — APPROVED** [review]

File: Applications/Spire/Include/Spire/Ui/SecurityListItem.hpp [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#ifndef SPIRE_SECURITY_LIST_ITEM_HPP
+#define SPIRE_SECURITY_LIST_ITEM_HPP
+#include <QWidget>
+#include "Nexus/Definitions/SecurityInfo.hpp"
```
Comment (kranar) [inline Applications/Spire/Include/Spire/Ui/SecurityListItem.hpp:4]: Include the utility header ```Ui.hpp```.

File: Applications/Spire/Source/Ui/SecurityListItem.cpp [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    : QWidget(parent),
+      m_security_info(std::move(security_info)) {
+  auto layout = new QVBoxLayout(this);
+  layout->setContentsMargins({});
+  layout->setSpacing(0);
+  auto value_container_layout = new QHBoxLayout();
+  value_container_layout->setContentsMargins({});
+  value_container_layout->setSpacing(0);
+  auto value_label = make_label(
+      QString::fromStdString(ToString(m_security_info.m_security)));
```
Comment (kranar) [inline Applications/Spire/Source/Ui/SecurityListItem.cpp:57]: Indentation.

File: Applications/Spire/Source/Ui/SecurityListItem.cpp [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+  value_container_layout->addWidget(value_label);
+  auto flag_icon_layout = new QVBoxLayout();
+  flag_icon_layout->setContentsMargins({});
+  flag_icon_layout->setSpacing(0);
+  flag_icon_layout->addStretch();
+  auto country_code = QString(GetDefaultCountryDatabase().
+    FromCode(m_security_info.m_security.GetCountry()).
+    m_threeLetterCode.GetData()).toLower();
+  auto flag_icon = new Icon(imageFromSvg(QString(":/Icons/flag_icons/%1.svg").
+      arg(country_code), FLAG_SIZE()), this);
```
Comment (kranar) [inline Applications/Spire/Source/Ui/SecurityListItem.cpp:69]: Indentation.

**Review by kranar — CHANGES_REQUESTED** [review]

**Pull request merged**